### ISSUE TITLE
Youtube Playlist downloading broken recently?

### ISSUE THREAD

**Issue by suntong** [rg3/youtube-dl#7621]
Did Youtube Playlist download broke just recently?

Today I tried to do Youtube Playlist downloading, but all my requests are failing:

```
 youtube-dl --continue --write-description 'https://www.youtube.com/playlist?list=PLsNCxZ6-KRGp3o6s9KvpUXNLTZQUroYmn'  
 youtube-dl --continue --write-description 'https://www.youtube.com/playlist?list=PLsNCxZ6-KRGprnQFXmwQ187NhcWuRWIHO'
 youtube-dl  --continue --write-description 'https://www.youtube.com/playlist?list=PLsNCxZ6-KRGrp7B24v-iKdwrskvxo6aIx'

$ youtube-dl -4 --continue --write-description 'https://www.youtube.com/playlist?list=PLsNCxZ6-KRGrUzInCfDVY2R5LBsV681eZ'
[youtube:playlist] PLsNCxZ6-KRGrUzInCfDVY2R5LBsV681eZ: Downloading webpage
ERROR: Unable to extract title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

```

What's interesting is the last one, which I was able to download successfully just several days ago. 

Can anyone confirm it please? 
Thanks

PS, my youtube-dl:

```
$ apt-cache policy youtube-dl
youtube-dl:
  Installed: 2015.02.28-1
  Candidate: 2015.02.28-1
  Version table:
 *** 2015.02.28-1 0
        500 http://azure.archive.ubuntu.com/ubuntu/ vivid/universe amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by phihag** [comment]
You are using an outdated version of youtube-dl (yours is about a hundred versions too old). Update to the latest version (if you don't know which is the latest, here it is: https://yt-dl.org/latest/version , 2015.11.23 as of writing). See [our FAQ](https://yt-dl.org/update) if you need help updating.

**Comment by suntong** [comment]
Ops, sorry for the noise. 

One think I'm not clear from the FAQ -- 
Does youtube-dl depends on any other packages? I got the following after `sudo apt-get remove -y youtube-dl`. Are none of them necessary? I did removed them all. Should I keep any of them? 

Thanks


% apt-get autoremove
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages will be REMOVED:
  liba52-0.7.4 libav-tools libavcodec56 libavdevice55 libavfilter5 libavformat56
  libavresample2 libavutil54 libdca0 libdirectfb-1.2-9 libdv4 libdvdnav4 libdvdread4 libfaad2
  libgif4 liblcms2-2 liblircclient0 liblua5.2-0 libmad0 libmpg123-0 libpostproc52
  libquvi-scripts libquvi7 libsmbclient libswscale3 libvpx1 libxss1 mplayer2 rtmpdump
0 upgraded, 0 newly installed, 29 to remove
